test/helpers.js

```diff
@@ -359,3 +359,19 @@ export const itReturnsErrorForUnverifiedVendor = ({
     });
   });
 };
+
+export const itDoesNotReturnSensitiveData = (data) => {
```

Comment:
This contains only expect assertions and used within expect statements, so it is better to make the naming start with expect...
```suggestion
export const expectResponseToExcludeSensitiveVendorData = (data) => {
```